sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -577,11 +583,14 @@ def add_message(self, message):
         :rtype: None
         :raises: :class: ~azure.servicebus.exceptions.MessageSizeExceededError, when exceeding the size limit.
         """
+
```

Comment:
we need to update the mypy, docstring accordingly, eg:
```python
    def add_message(self, message):
        # type: (Union[ServiceBusMessage, dict]) -> None
        :param message: The message to be added to the batch.
        :type message: Union[~azure.servicebus.ServiceBusMessage, dict]
```

Reply:
same comment applies to other methods which now accept dict-representation.

Reply:
Same as above: https://github.com/Azure/azure-sdk-for-python/pull/14807#discussion_r516872709